src/til/ut_til/RunLengthEncodingTests.cpp

```diff
@@ -0,0 +1,723 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "precomp.h"
+
+#include "til/rle.h"
+
+using namespace WEX::Common;
+using namespace WEX::Logging;
+using namespace WEX::TestExecution;
+
+class RunLengthEncodingTests
+{
+    TEST_CLASS(RunLengthEncodingTests);
+
+    TEST_METHOD(ConstructDefaultLength)
+    {
+        til::rle<unsigned int> rle(86, 9);
+        auto foo = rle.begin();
+    }
+
+    TEST_METHOD(ConstructVerySmall)
+    {
+        const til::rle<unsigned short, unsigned char> rle(12, 37);
+    }

```

Comment:
Do we need to do any other verifications here?

Reply:
Not... really. I added one anyway but it was really about making sure the template works for this case. Do you have another idea of what else would be appropriate here?